Makefile

```
@@ -96,21 +115,28 @@ test: check_docker ## Run 'deno task test' in the api container
 	@echo "=> Running 'deno task test' within api container ..."
 	@docker exec -it whereis-api-v0 deno task test
 
+initkey: check_docker ## Run 'deno task initkey' in the api container. Usage: make initkey key=xxx user=xxx
```

checkmake (0.2.2)

[warning] Target body for "initkey" exceeds allowed length of 5 (7).

(maxbodylength)